.github/workflows/php-lint.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - trunk
       - 'release/**'
+      - 'feature/**'
```

Comment:
Same here, we don't need to run this workflow on every push to a feature branch:
```suggestion
```